src/views/router_views/confirmation_submission/confirmation_submission.njk

```diff
@@ -25,5 +25,5 @@
 
     <h2 class="govuk-heading-m">File more package accounts</h2>
     <p class="govuk-body"><a href={{ prefixedUrls.UPLOAD }}>File more package accounts for {{companyProfile.companyName}} ({{companyProfile.companyNumber}})</a></p>
-    <p class="govuk-body"><a href={{ prefixedUrls.HOME }}>File more package accounts for a different company</a></p>
+    <p class="govuk-body"><a href={{ prefixedUrls.COMPANY_SEARCH }}>File more package accounts for a different company</a></p>
```

Comment:
Why does this add a new global template variable `prefixedUrls` when the `Urls` object (which are the prefixed urls) already exists in the BaseViewData?

Either remain consistant which other pages and use the `Urls` variable, or update all other pages to use the new `prefixedUrls` global.